### PR TITLE
add amount reimbursed field

### PR DESCRIPTION
**Pull request by victorzheng02**
See https://github.com/WATonomous/wato_finance_system/issues/48

### CODE REVIEW

File: backend/server.js [hunk 1]
```diff
@@ -43,7 +43,6 @@ app.use('/googlegroups', groupRouter)
 
 app.listen(port, async () => {
     console.log(`Server is running on port: ${port}`)
-    await updateGroup()
```
Comment (ansonjwhe) [inline backend/server.js:46]: what is the purpose of removing this

File: frontend/src/components/CreateTicketModal.js [hunk 2]
```diff
@@ -28,6 +28,7 @@ export function CreateTicketModal({ isOpen, onClose }) {
     const { control, register, handleSubmit } = useForm()
     const auth = useAuth(app)
 
+    console.log(auth.currentUser)
```
Comment (ansonjwhe) [inline frontend/src/components/CreateTicketModal.js]: do we need to keep this console log

**Review by ansonjwhe — APPROVED** [review]
overall LGTM, some unrelated changes to the original issue should be addressed

File: backend/server.js (hunk 1 above)
Comment (victorzheng02) [reply backend/server.js:46]: I was the one who originally added it while testing, honestly it makes sense to keep it so im fine with leaving it there

**Pull request merged**